insights/__init__.py

```diff
@@ -160,33 +159,38 @@ def run(component=None, root=None, print_summary=False,
         import logging
         p = argparse.ArgumentParser()
         p.add_argument("archive", nargs="?", help="Archive or directory to analyze")
-        p.add_argument("-p", "--plugins", default=[], nargs="*",
-                       help="package(s) or module(s) containing plugins to run.")
+        p.add_argument("-p", "--plugins", default="", help="package(s) or module(s) containing plugins to run.")
```

Comment:
You might consider adding that this parameter is a comma separated list with no spaces in the help.  I  tried `-p plugins1,  plugins2` and it failed trying to use plugins2 as the archive.

Reply:
Good idea. Done.